engine/lib/deprecated-4.2.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Bundle all functions which have been deprecated in Elgg 4.2
+ */
+
+/**
+ * Render a menu item (usually as a link)
+ *
+ * @param \ElggMenuItem $item The menu item
+ * @param array         $vars Options to pass to output/url if a link
+ *
+ * @return string
+ * @since 1.9.0
+ * @deprecated 4.2 use elgg_view('navigation/menu/elements/item/url')
```

Comment:
i believe we never add the version number here...

Reply:
this should be provided by the PHPDoc https://docs.phpdoc.org/guide/references/phpdoc/tags/deprecated.html